work-process/scrum.md

```diff
@@ -58,11 +58,23 @@ This is also the time to bring up any other comments about the work process in g
 
 We look at how many Story Points we assigned, how many were done and the distribution between the departments and types of User Stories.
 
-We go through the [cancellation reasons of our VIP users](https://github.com/niteoweb/support/blob/master/EBN/cancellation-analysis.md) (Agency plan and above). Since they are our primary customers, we want to know why they're leaving and check if there are any actionable reasons.
+The Scrum Master will create a new [retrospective issue](https://github.com/niteoweb/operations/issues/new?template=retrospective.md&title=Retrospective%20for%20Sprint%20#) before each retrospective. This issue has a template that serves as a checklist for the meeting and helps the Scrum Master write up a brief report afterwards.
+
+
+## Orientation Meeting
```

Comment:
`Dharma Initiative: Orientation Meeting` 😄 

P.S. Sorry I could not help myself ^^ 😄

Reply:
😄